source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -1,3 +1,85 @@
+.. _accessors:
+
 =========
 Accessors
 =========
+
+This section contain definisions of classes that implement :ref:`accessor`
+concept.
+
+------------------------------
+Requirements on accessor types
+------------------------------
+
+Each implementation of :ref:`accessor` concept in |dal_short_name| shall follow
+next requirements:
+
+1. Define single :term:`format of the data <Data format>` of the
+   accessor. Every single accessor type shall provide only one :term:`format of
+   the data <Data format>` returned from it or put inside it.
```

Comment:
```suggestion
1. Define a single :term:`format of the data <Data format>` for the
   accessor. Every single accessor type shall return and use
   only one :term:`format of the data <Data format>` .
```
`use` might not be the correct word, but `put inside it` is unclear
why not use just `data format` instead of the seonc `format of the data`?